src/sdk/main/src/Status.cc

```diff
@@ -25,914 +25,923 @@ namespace Hedera
+  { Status::SCHEDULE_PENDING_EXPIRATION,                                    "SCHEDULE_PENDING_EXPIRATION"                        },
+  { Status::CONTRACT_IS_TOKEN_TREASURY,                                     "CONTRACT_IS_TOKEN_TREASURY"                         },
+  { Status::CONTRACT_HAS_NON_ZERO_TOKEN_BALANCES,                           "CONTRACT_HAS_NON_ZERO_TOKEN_BALANCES"               },
+  { Status::CONTRACT_EXPIRED_AND_PENDING_REMOVAL,                           "CONTRACT_EXPIRED_AND_PENDING_REMOVAL"               },
+  { Status::CONTRACT_HAS_NO_AUTO_RENEW_ACCOUNT,                             "CONTRACT_HAS_NO_AUTO_RENEW_ACCOUNT"                 },
+  { Status::PERMANENT_REMOVAL_REQUIRES_SYSTEM_INITIATION,                   "PERMANENT_REMOVAL_REQUIRES_SYSTEM_INITIATION"       },
+  { Status::PROXY_ACCOUNT_ID_FIELD_IS_DEPRECATED,                           "PROXY_ACCOUNT_ID_FIELD_IS_DEPRECATED"               },
+  { Status::SELF_STAKING_IS_NOT_ALLOWED,                                    "SELF_STAKING_IS_NOT_ALLOWED"                        },
+  { Status::INVALID_STAKING_ID,                                             "INVALID_STAKING_ID"                                 },
+  { Status::STAKING_NOT_ENABLED,                                            "STAKING_NOT_ENABLED"                                },
+  { Status::INVALID_PRNG_RANGE,                                             "INVALID_PRNG_RANGE"                                 },
+  { Status::MAX_ENTITIES_IN_PRICE_REGIME_HAVE_BEEN_CREATED,                 "MAX_ENTITIES_IN_PRICE_REGIME_HAVE_BEEN_CREATED"     },
+  { Status::INVALID_FULL_PREFIX_SIGNATURE_FOR_PRECOMPILE,                   "INVALID_FULL_PREFIX_SIGNATURE_FOR_PRECOMPILE"       },
+  { Status::INSUFFICIENT_BALANCES_FOR_STORAGE_RENT,                         "INSUFFICIENT_BALANCES_FOR_STORAGE_RENT"             },
+  { Status::MAX_CHILD_RECORDS_EXCEEDED,                                     "MAX_CHILD_RECORDS_EXCEEDED"                         },
+  { Status::INSUFFICIENT_BALANCES_FOR_RENEWAL_FEES,                         "INSUFFICIENT_BALANCES_FOR_RENEWAL_FEES"             },
+  { Status::TRANSACTION_HAS_UNKNOWN_FIELDS,                                 "TRANSACTION_HAS_UNKNOWN_FIELDS"                     },
+  { Status::ACCOUNT_IS_IMMUTABLE,                                           "ACCOUNT_IS_IMMUTABLE"                               },
+  { Status::ALIAS_ALREADY_ASSIGNED,                                         "ALIAS_ALREADY_ASSIGNED"                             },
+  { Status::INVALID_METADATA_KEY,                                           "INVALID_METADATA_KEY"                               },
+  { Status::MISSING_TOKEN_METADATA,                                         "MISSING_TOKEN_METADATA"                             },
+  { Status::MISSING_SERIAL_NUMBERS,                                         "MISSING_SERIAL_NUMBERS"                             }
```

Comment:
What `clang-format` do you have installed? I don't think the `Status`es you added here should create all these changes

Reply:
clang-format version 18.1.1

Reply:
Pipeline is: 
```
  - name: Run Clang-Format
        uses: jidicula/clang-format-action@v4.11.0
        with:
          clang-format-version: "17"
          check-path: "src"
```
Maybe I should downgrade.